tools/binlog/binlog-slave-client.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Binlog Slave Client 用户文档
+category: tools
+---
+
+# Binlog Slave Client 用户文档
+
+目前 Drainer 提供了多种输出方式，包括 mysql、tidb、file 等。但是用户往往有一些自定义的需求，比如输出到 Elasticsearch、Hive 等，这些需求 Drainer 现在还没有实现，因此 Drainer 增加了输出到 Kafka 的功能，将 binlog 数据解析后按一定的格式再输出到 Kafka 中，用户编写代码从 Kafka 中读出数据再进行处理。
+
+## 配置 Kafka Drainer 
+
+修改 Drainer 的配置文件，设置输出为 Kafka，相关配置如下：
+
+```
+[syncer]
+db-type = "kafka"
+
+[syncer.to]
+# Kafka 地址
+kafka-addrs = "127.0.0.1:9092"
+# Kafka 版本号
+kafka-version = "0.8.2.0"
+```
+
+##  配置 File Drainer
```

Comment:
we should remove it, `file` and `kafka` type have different protocol definition 😢

Reply:
removed in https://github.com/pingcap/docs-cn/pull/1264/commits/d15ab05fa2fd0eca2ac902eba178bf5644e89a3c